src/main/java/seedu/weme/model/statistics/StatsManager.java

```diff
@@ -35,49 +36,41 @@ public StatsManager(Stats stats) {
     }
 
     //============= Like Data ====================================
-    /**
-     * Replaces the contents of the like data with {@code likeData}.
-     */
+
+    public void addDefaultLikeData(Meme meme) {
+        likeManager.initMemeLikeCount(meme);
```

Comment:
rename this to add default meme like count?

Reply:
Any reason?